lib/embed/embed.dart

```diff
@@ -330,10 +331,16 @@ class PlaygroundMobile {
     }
   }
 
+  void bindSplitter() {
```

Comment:
How about make this private, and change the name to something that reflects that it's used to sync the main toolbar width with the editing area's width?